annsa/tests/data_folder/rocky_flats_spectra.spe

```diff
@@ -0,0 +1,1050 @@
+$SPEC_ID:

```

Comment:
are you allowed to share this spe file online?

Reply:
Yes, sharing the spectra online are fine.